lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -905,17 +918,18 @@ def run(field, handle_metadata_errors=True):
     """
     Convert a NIMROD field to an Iris cube.
 
-    Args:
-
-        * field - a :class:`~iris.fileformats.nimrod.NimrodField`
-
-        * handle_metadata_errors - Set to False to omit handling of known meta-data deficiencies
-                                   in Nimrod-format data
-
-    Returns:
+    Args
+    ----
+    field: :class:`~iris.fileformats.nimrod.NimrodField`
```

Comment:
Not sure why in preview this has a "-" on the end of the line?

Reply:
Happens a few times - I think it's expecting you to follow the line with a description? Looks particularly odd where it happens after `**kwargs`

Reply:
This is expected behaviour I believe.  When defining the Args you can:

* define the return type
* define the description of the return
* do both

The one you queried just does the first, hence the hyphen when the description would be but there is none defined.  For me, this PR was about fixing the formatting and doing some light improvements, someone with more familiarity with Iris will need to review and improve the quality of the docstrings generally.